pandas/tests/indexing/test_at.py

```diff
@@ -97,6 +99,18 @@ def test_at_setitem_categorical_missing(self):
 
         tm.assert_frame_equal(df, expected)
 
+    @pytest.mark.parametrize("row", (to_datetime("2019-01-01"), "2019-01-01"))
```

Comment:
Could you please check the whole DataFrame with tm.assert_frame_equal and use Timestamp() instead too?